pepper-apis/dsm-core/src/main/resources/logback.xml

```diff
@@ -1,12 +1,28 @@
 <configuration>
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
         <encoder>
             <pattern>%d{HH:mm:ss.SSS} %X{LogBreadCrumb} %X{X-Forwarded-For} C:%X{ClientId} S:%X{UserId} [%thread] %-5level %logger{36} %msg%n</pattern>
         </encoder>
+        <filter class="ch.qos.logback.classic.filter.LevelFilter">
+            <level>ERROR</level>
+            <onMatch>DENY</onMatch>
```

Comment:
Does the DENY here mean do not log "ERROR"s to stdout?

Reply:
Right. Happy to change that, but it seemed like we should not duplicate?